lib/rules/block-spacing.js

```diff
@@ -102,9 +102,18 @@ module.exports = {
 
             // Check.
             if (!isValid(openBrace, firstToken)) {
+                let loc = openBrace.loc.start;
```

Comment:
This should be just `openBrace.loc`

Reply:
I have kept the loc same as current for `missing` so this is the current (`master`) location. (just refactored it.)

should I change this then ?

Reply:
Yes, we're making that change in other rules as well to avoid reporting zero-length ranges where possible.

Reply:
cool ok 👍

Reply:
Done